testah-junit/src/test/java/org/testah/driver/web/element/WebElementWrapperV1Test.java

```diff
@@ -3,33 +3,21 @@
 import org.junit.*;
 import org.openqa.selenium.By;
 import org.testah.TS;
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-import org.openqa.selenium.By;
-import org.testah.TS;
 import org.testah.framework.cli.Params;
 
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.notNullValue;
-
-
 import static org.hamcrest.Matchers.*;
```

Comment:
no import *?

Reply:
not great practice the import star, but this is intellij not me. I think its much better with direct import.